src/libAtomVM/mailbox.c

```diff
@@ -367,14 +367,14 @@ bool mailbox_peek(Context *c, term *out)
     return true;
 }
 
-void mailbox_remove_message(Mailbox *mbox, Heap *heap)
+MailboxMessage *mailbox_take_message(Mailbox *mbox)
 {
     // This is called from OP_REMOVE_MESSAGE opcode, so we cannot make any
```

Comment:
Should we move the comment down to the mailbox_remove_message function?  And maybe leave a comment that this function (mailbox_take_message) might be used for third party drivers?

Reply:
Better yet, if it is intended for third party drivers it should have proper doxygen style documentation comments, so it can be published with the API documentation online.

Reply:
I would keep that comment there, it explains what we do on the next lines.